deeppavlov_agent/base_settings.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from importlib import import_module
+
+from .core.db import DataBase
+from .core.state_manager import StateManager
+from .core.workflow_manager import WorkflowManager
+from .state_formatters.output_formatters import (http_api_output_formatter,
+                                                 http_debug_output_formatter)
+
+# Default parameters
+BASE_PARAMETERS = {
+    'debug': True,
+    'state_manager_class': StateManager,
+    'workflow_manager_class': WorkflowManager,
+    'db_class': DataBase,
+    'pipeline_config': 'pipeline_conf.json',
+    'db_config': 'db_conf.json',
+    'overwrite_last_chance': None,
+    'overwrite_timeout': None,
+    'formatters_module': None,
+    'connectors_module': None,
+    'response_logger': True,
+    'time_limit': 0,
+    'output_formatter': http_api_output_formatter,
+    'debug_output_formatter': http_debug_output_formatter,
+    'port': 4242,
+    'telegram_token': '',
+    'telegram_proxy': '',
+}
+
+# Replasing constants with ones from user settings
+
+def setup_parameter(name, user_settings):
+    res = None
+    if user_settings:
+        res = getattr(user_settings, name, None)
+    if res is None:
+        res = BASE_PARAMETERS[name]
+    return res
```

Comment:
```suggestion
    return getattr(user_settings, name) if hasattr(user_settings, name) else BASE_PARAMETERS[name]
```
otherwise you'll get unexpected behavior if a parameter is `None`

Reply:
Может рвануть